spec/lib/model/request_spec.rb

```diff
@@ -128,7 +128,7 @@
 
     describe 'returns false if repo data is not available' do
       let(:pull_request) { PullRequest.new }
-      it { request.same_repo_pull_request?.should be_falsy }
+      it { request.same_repo_pull_request?.should be_falsey }
```

Comment:
`be_falsy` is an alias for `be_falsey`.

Personally, I prefer `falsy`, but in Ruby (and the majority of this repo), `falsey` is used.